packages/material-ui-system/src/createStyled.d.ts

```diff
@@ -76,7 +76,7 @@ export type PropsOf<C extends keyof JSX.IntrinsicElements | React.JSXElementCons
 
 export type Overwrapped<T, U> = Pick<T, Extract<keyof T, keyof U>>;
 
-type JSXInEl = JSX.IntrinsicElements;
+export type JSXInEl = JSX.IntrinsicElements;
```

Comment:
Let's just remove this type. Abbreviations are ok for local development but not for merging code that others work with.

Reply:
Done